hive-site.xml

```diff
@@ -0,0 +1,358 @@
+  <value>Sqoop12345</value>
+</property>
+-->
+
+<!--
+<property>
+  <name>javax.jdo.option.ConnectionURL</name>
+  <value>jdbc:postgresql://172.28.199.31:5432/postgres</value>
+</property>
+
+<property>
+  <name>javax.jdo.option.ConnectionDriverName</name>
+  <value>org.postgresql.Driver</value>
+</property>
+
+<property>
+  <name>javax.jdo.option.ConnectionUserName</name>
+  <value>sqoop</value>
+</property>
+
+<property>
+  <name>javax.jdo.option.ConnectionPassword</name>
+  <value>Sqoop12345</value>
+</property>
+-->
+
+<property>
+  <name>javax.jdo.option.ConnectionURL</name>
+  <!--value>jdbc:mysql://localhost/upstream_hive?useLegacyDatetimeCode=false&amp;serverTimezone=Europe/Budapest</value-->
+  <value>jdbc:mysql://localhost/upstream_hive</value>
+  <description>JDBC connect string for a JDBC metastore</description>
+</property>
+
+<property>
+  <name>hive.metastore.try.direct.sql</name>
+  <value>true</value>
+</property>
+
+<property>
+  <name>javax.jdo.option.ConnectionDriverName</name>
+  <value>com.mysql.jdbc.Driver</value>
+</property>
+
+<property>
+  <name>javax.jdo.option.ConnectionUserName</name>
+  <value>upstream_hive</value>
+</property>
+
+<property>
+  <name>javax.jdo.option.ConnectionPassword</name>
+  <value>hive</value>
+</property>
+
+<property>
+  <name>hive.server2.enable.doAs</name>
+  <value>false</value>
+  <description></description>
+</property>
+
+<property>
+  <name>hive.server2.enable.impersonation</name>
+  <value>false</value>
+  <description></description>
+</property>
+
+<property>
+  <name>dfs.namenode.acls.enabled</name>
+  <value>false</value>
+</property>
+
+<!-- FAIR SCHEDULER -->
+
+<!--
+<property>
+  <name>mapred.jobtracker.taskScheduler</name>
+  <value>org.apache.hadoop.mapred.FairScheduler</value>
+</property>
+
+<property>
+  <name>mapred.fairscheduler.allocation.file</name>
+  <value>/mnt/samsung/dev-tools/conf/allocations.xml</value>
+</property>
+
+<property>
+  <name>mapred.fairscheduler.poolnameproperty</name>
+  <value>pool.name</value>
+</property>
+
+<property>
+  <name>pool.name</name>
+  <value>${user.name}</value>
+</property>
+-->
+
+<!-- These following lines are needed to use ACID features -->
+<!-- BEGIN -->
+
+<!--
+<property>
+  <name>hive.enforce.bucketing</name>
+  <value>true</value>
+</property>
+<property>
+  <name>hive.support.concurrency</name>
+  <value>true</value>
+</property>
+<property>
+  <name>hive.exec.dynamic.partition.mode</name>
+  <value>nonstrict</value>
+</property>
+<property>
+  <name>hive.txn.manager</name>
+  <value>org.apache.hadoop.hive.ql.lockmgr.DbTxnManager</value>
+</property>
+<property>
+  <name>hive.lock.manager</name>
+  <value>org.apache.hadoop.hive.ql.lockmgr.DbLockManager</value>
+</property>
+<property>
+  <name>hive.compactor.initiator.on</name>
+  <value>true</value>
+</property>
+<property>
+  <name>hive.compactor.worker.threads</name>
+  <value>2</value>
+</property>
+-->
+<!-- END -->
+
+</configuration>
```

Comment:
if this is for testing please added as a resource.

Reply:
Removed